### PR TITLE
#2641. Add wildcards do not shadow tests

### PR DESCRIPTION
**Pull request by sgrekhov**

### CODE REVIEW

File: LanguageFeatures/Wildcards/wildcards_do_not_shadow_A01_t04.dart [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+///     var _ = 'local';
+///
+///     _ = 'assign';
+///   }
+/// }
+/// ```
+// This program is valid and assigns to the field, not the local.
+///
+/// @description Checks that a local variable named `_` doesn't shadow a
+/// top-level ыуееук named `_` but a class field does.
```
Comment (eernstg) [inline LanguageFeatures/Wildcards/wildcards_do_not_shadow_A01_t04.dart:20]: Perhaps? ;-)
```suggestion
/// top-level method named `_` but a class field does.
```

**Review by eernstg — COMMENTED** [review]
Looks good, just one typo!

**Comment by sgrekhov** [comment]
Really :) Fixed.

**Review by eernstg — APPROVED** [review]
LGTM

**Pull request merged**